src/families/ethereum/modules/send.js

```diff
@@ -99,13 +99,19 @@ const send: ModeModule = {
     }
   },
 
-  fillDeviceTransactionConfig({ status: { amount } }, fields) {
+  fillDeviceTransactionConfig({ transaction, status: { amount } }, fields) {
     if (!amount.isZero()) {
       fields.push({
         type: "amount",
         label: "Amount",
       });
     }
+    if (transaction.data?.length) {
+      fields.push({
+        type: "data",
+        label: "Data",
+      });
```

Comment:
do you think it's possible to use a `type: text` instead with the value that would display on device anyway? to me it's not to translate because device is in English.
and on long term future we want to kill all the extra type and try to only have type:text.

Reply:
yes { type: "text", "label": "Data", value: "YES/NO" }

Reply:
the only problem i see with this technique is regarding translation. maybe value should be an i18nkey

Reply:
TBD:
- is this the string that display on device? let's try to be ISO
- we don't need translation precisely because we need to be ISO and device is currently English only. if device in future gets translated, we'll reconsider this, but right now it's a choice done for similar parts.